devtools/conda-envs/test_env.yaml

```diff
@@ -10,7 +9,6 @@ dependencies:
     # Testing
   - pytest
   - pytest-cov
-  - codecov
```

Comment:
This is no longer needed on the user side - the official action handles uploading itself